src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,107 @@
+            customtr.innerHTML = '  <td></td><td colspan="2"><div class="help"><div class="custom-help-button">'+
+            '<div class="custom-help-button-save" onclick="save_custom_help(this)" >Save</div><div class="custom-help-button-edit" onclick="edit_custom_help(this)" >Edit</div>'+
+            '</div><div class="custom-help-text">Loading1...</div></div></td><td></td>';
+            $(customtr).addClassName("help-area");
+            $(customtr).addClassName("custom-help");
+            $(thistr).insert({after:customtr});
+            // Modification of the inner structure of the created element
+            $(customtr).down().next().down().down().down().next().style.display = "block";
+            // Location for the default help
+            var div = $(tr).down().next().down();
+            // Location for the custom help
+            var div2 = $(customtr).down().next().down().down().next()
+            var div3 = $(customtr).down().next().down();
+            div.style.display = "block";
+            div3.style.display = "block";
+
+            // make it visible
+            new Ajax.Request(this.getAttribute("helpURL"), {
+                method : 'get',
+                onSuccess : function(x) {
+                    var from = x.getResponseHeader("X-Plugin-From");
+                    div.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                    layoutUpdateCallback.call();
+                },
+                onFailure : function(x) {
+                    div.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                    layoutUpdateCallback.call();
+                }
+            });
+            
+            // Creation of unique id
+            var customHelpClassName = this.getAttribute("helpURL").replace(rootURL+"/", "").replace("/", ".");
+            while(customHelpClassName.indexOf("/") + 1){
+                customHelpClassName = customHelpClassName.replace("/", ".")
+            }
+
+            var customurl = rootURL + "/helpmanager/get?class="+customHelpClassName;
+                $(div2).setAttribute("customHelpUrl", rootURL+"/helpmanager/updateHelpInfo");
+                $(div2).setAttribute("className", customHelpClassName);
+
+            new Ajax.Request(customurl, {
+                method : 'get',
+                onSuccess : function(x) {
+                    var from = x.getResponseHeader("X-Plugin-From");
+                    div2.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                    layoutUpdateCallback.call();
+                },
+                onFailure : function(x) {
+                    div2.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                    layoutUpdateCallback.call();
+                }
+            });
+
+        } else {
+            var div = $(findFollowingTR(tr, "help-area")).down().next().down();
```

Comment:
@michaelneale I took this from hudson-behavior.js here. Is it fine then?
https://github.com/jenkinsci/jenkins/blob/master/war/src/main/webapp/scripts/hudson-behavior.js#L729